src/client.js

```diff
@@ -416,6 +416,58 @@ class Client {
     prom.then(cb)
   }
 
+    /**
+   * Searches for messages that include the search string according to `opts`.
+   * @param {string} [searchString] string to match messages against
+   * @param {Object} [opts] 
+   * @param {number} [opts.olderThan] timestamp in epoch time. we want to search through messages that are *older* than this ts
+   * @param {number} [opts.newerThan] timestamp in epoch time. we want to search through messages that are *newer* than this ts
+   * @param {number} [opts.amount] amount of messages to be search through
+   * @param {string} [opts.channel] channel to get messages from. defaults to currently focused channel
+   * @param {Cabal} [cabal=this.currentCabal]
+   */
+  searchMessages(searchString, opts, cabal = this.currentCabal) {
+    return new Promise((resolve, reject) =>
+      {
+        if(!searchString && searchString != "") {
+          reject('search string must be set')
+        }
```

Comment:
```suggestion
        if (!searchString && searchString !== "") {
          return reject('search string must be set')
        }
```

Reply:
Does the `return`  here have a function, or did you add it to clarify that the method exits there?

Reply:
iirc i think execution will continue without it? *goes to check if i'm actually mistaken lol*

Reply:
https://gist.github.com/cblgh/6b98da32d30550e181305fe6e6bd61f2 looks like i'm not mistaken, try this gist

Reply:
Huh, I didn't know that :O. Not sure why I expected otherwise since `reject` is just a function call